region/crateria/central.json

```diff
@@ -365,6 +534,23 @@
                       "requires": ["h_canDestroyBombWalls"]
                     }
                   ]
+                },
+                {
+                  "name": "G-Mode through Bomb Blocks",
```

Comment:
You may want to put this in a link from 1->4, as it could be used to get to the bottom right door without a wall jump (if that is ever removed from the game.) going from 4->5 is free, but 5->4 could require wall jumps. A similar strat could be added for 2->4

Reply:
Done, thanks!